### PR TITLE
Linking existing harvests to a planting

### PR DESCRIPTION
**Pull request by Br3nda**
Links to something with matching crop.

TODO: Unlinking later.

### CODE REVIEW

**Comment by maco** [comment]
Do you want to do unlinking as a separate PR?

File: app/views/harvests/_planting.haml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+- if @harvest.planting
+  = link_to "#{@harvest.planting.crop.name} planted on #{@harvest.planting.planted_at}",
```
Comment (pozorvlak) [inline app/views/harvests/_planting.haml]: We'll have to internationalise this at some point.

File: app/views/harvests/_planting.haml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    planting_path(@harvest.planting)
+  in
+  = link_to @harvest.planting.garden, garden_path(@harvest.planting.garden)
+- elsif @matching_plantings && @harvest.owner == current_member
+  Is this from one of these plantings?
+  = form_for(@harvest) do |f|
+    - @matching_plantings.each do |planting|
+      = f.radio_button :planting_id, planting.id
+      = label_tag(:planting, planting)
+      %br/
```
Comment (pozorvlak) [inline app/views/harvests/_planting.haml]: Is `br` the Right Thing here? We seem to use `.field` to get each field on a new line in `account_types/_form.html.haml`.

Comment (Br3nda) [inline app/views/harvests/_planting.haml]: changed to .field.

**Comment by pozorvlak** [comment]
```
Failures:
  1) forums as an admin user adding a forum
     Failure/Error: expect(current_path).to eq forum_path(Forum.last)
     
     ActionController::UrlGenerationError:
       No route matches {:action=>"show", :controller=>"forums", :id=>nil} missing required keys: [:id]
     # ./spec/features/admin/forums_spec.rb:42:in `block (3 levels) in <top (required)>'
```

That's weird...

**Pull request merged**